lib/query.js

```diff
@@ -57,3 +102,111 @@ export const getValue = node => {
 
 export const getNumerator = (node) => isNeg(node) ? getNumerator(node.args[0]) : node.args[0]
 export const getDenominator = (node) => isNeg(node) ? getDenominator(node.args[0]) : node.args[1]
+
+// TODO: handle multivariable polynomials
+// Get degree of a polynomial term
+// e.g. 6x^2 -> 2
+export const getExponent = (node) => {
+    if (isNumber(node)) {
+        return 0
+    } else if (isIdentifier(node) || isPolynomial(node)){
+        return 1
+    } else if (isPow(node)) {
+        return getValue(node.args[1])
+    } else if (isMul(node)){
+        return getExponent(node.args[1])
+    } else if (isNeg(node)) {
+        const variable = node.args[0]
+        return getExponent(variable.args[1])
+    } else {
+        return null
+    }
+}
+
+export const getCoefficient = (node) => {
+    if (isNumber(node)) {
+        return node
+    } else if (isIdentifier(node) || isPow(node)) {
+        return build.numberNode(1)
+    } else if (isNeg(node)) {
+        const result = build.applyNode('neg', [getCoefficient(node.args[0])])
+        result.wasMinus = node.wasMinus
+        return result
+    } else if (isMul(node)) {
+        const numbers = node.args.filter(arg => isNumber(arg) || isConstantFraction(arg))
+        if (numbers.length > 1) {
+            return build.applyNode('mul', numbers)
+        } else if (numbers.length > 0) {
+            return numbers[0]
+        } else {
+            return build.numberNode(1)
```

Comment:
`build.number('1')`... I need to update `buildNumber`/`number` to convert numbers to strings.